### ISSUE TITLE
Broken relation when deleting node

### ISSUE THREAD

**Issue by matheusgomesms** [openstreetmap/iD#9535]
### URL

https://www.openstreetmap.org/relation/15521782/history

### How to reproduce the issue?

First, sorry if this was reported before; I searched through the issues and could not find the same problem reported.

When there is a relation of `type=enforcement`, roles `from` and `to` are supposed to be `node`, per wiki. However, when on iD a node part of an enforcement relation is deleted, iD throws no warning and the relation is broken. You can check it in this area, with [this relation](https://www.openstreetmap.org/relation/15521782/history). Try to delete [this node](https://www.openstreetmap.org/node/10719723254) and the relation is broken.

### Which deployed environments do you see the issue in?

Released version at openstreetmap.org/edit

### What version numbers does this issue effect?

2.25.1

**Comment by sun-geo** [comment]
I agree - see older duplicate request: https://github.com/openstreetmap/iD/issues/5846
